admin/tabs/activity/clienterrors/ClientErrorsModel.js

```diff
@@ -178,21 +231,22 @@ export class ClientErrorsModel {
         let newStart = start[dir](incr),
             newEnd = end[dir](incr);
 
-        if (newEnd.diff(today) > 0 || toToday) {
-            newStart = today.subtract(Math.abs(diff));
-            newEnd = today;
+        if (newEnd > tomorrow) {
+            newStart = tomorrow.subtract(Math.abs(diff));
+            newEnd = tomorrow;
         }
 
-        this.setStartDate(newStart);
-        this.setEndDate(newEnd);
+        this.startDate = newStart;
+        this.endDate = newEnd;
     }
 
     getParams() {
-        return {
-            startDate: this.startDate,
-            endDate: this.endDate,
-            username: this.username,
-            error: this.error
-        };
+        const {startDate, endDate} = this;
+        return {startDate, endDate};
```

Comment:
This can actually be a one liner: 
`return {startDate, endDate} = this`;

Reply:
FYI, I found this on SO which says doing it as a one-liner creates global vars:
https://stackoverflow.com/questions/41794086/es6-destructuring-within-a-return-statement